glusterfs/orchestrate/server.sls

```diff
@@ -0,0 +1,18 @@
+glusterfs.server.service:
+  salt.state:
+    - tgt: 'glusterfs:server'
+    - tgt_type: pillar
+    - queue: True
+    - sls: glusterfs.server.service
+    - batch: 1
```

Comment:
After a lot of time spent on testing Salts behaviour during orchestrate run, I prefer batch run over concurrent run